README.md

```diff
@@ -48,8 +48,8 @@ See https://github.com/agntcy/dir/releases
 ### Container images
```

Comment:
golang packages above are not working, lets cover those as well, valid paths are:
ttps://pkg.go.dev/github.com/agntcy/dir/api
ttps://pkg.go.dev/github.com/agntcy/dir/server
ttps://pkg.go.dev/github.com/agntcy/dir/client

Reply:
would be nice to have a badge on the top of the README to link to these

Reply:
let me add those common project badges in another PR